notebooks/for_scientists/index.clj

```diff
@@ -14,7 +14,7 @@
   ;; - are clay kinds executed outside of a clay environment, i.e. are they treated like 'comment's under 'normal' circumstances?
   ;; - Move general notes and examples out from the Cavity section
   ;; - should probably be split into two (the cavity part can be quite self-contained)
-  ,)
```

Comment:
the comma was there on purpose, so that parinfer does not move the closing paren :-)